src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import commands.Command;
+import parser.Parser;
+import storage.DataFile;
+import tasks.Task;
+import tasks.TaskList;
+import ui.Ui;
+import java.io.*;
```

Comment:
Should wildcard imports be used? Maybe it is better to list them out?

Reply:
yup, thx for pointing it out